pkgs/tools/filesystems/wipefreespace/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenv
+, lib
+, fetchurl
+, e2fsprogs
+, ntfs3g
+, xfsprogs
+, reiserfsprogs
+, reiser4progs
+, jfsutils
+, libuuid
+, texinfo
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
```

```suggestion
stdenv.mkDerivation rec {
```
redundant and causes tooling (`nixpkgs-hammering`) to complain